test/extended/util/framework.go

```diff
@@ -2102,6 +2102,24 @@ func DoesApiResourceExist(config *rest.Config, apiResourceName, group string) (b
 	return false, nil
 }
 
+func IsNamespaceExist(kubeClient *kubernetes.Clientset, namespace string) (bool, error) {
+	ns, err := kubeClient.CoreV1().Namespaces().Get(context.Background(), namespace, metav1.GetOptions{})
+	if err != nil {
+		if kapierrs.IsNotFound(err) {
+			e2e.Logf("%s namespace not found", namespace)
+			return false, nil
+		}
+		e2e.Logf("error accessing %s namespace: %v", namespace, err)
```

Comment:
I'd avoid logging the error since it's handled by the caller already.

Reply:
Done!